src/app/containers/MediaPlayer/index.jsx

```diff
@@ -48,27 +48,37 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
     return null; // this should be the holding image with an error overlay
   }
 
+  const hidePlaceholder =
+    typeof embedOverrides.showPlaceholder !== 'undefined' &&
```

Comment:
Could just `!embedOverrides.showPlaceholder` not work?

Reply:
Going over the logic again here, I don't think so - we want the placeholder to be shown unless it was explicitly overridden to be removed

eg:
`embedOverrides.showPlaceholder: true` -> placeholder shown
`embedOverrides.showPlaceholder: undefined` -> placeholder shown
`embedOverrides.showPlaceholder: false` -> placeholder hidden

Reply:
Made this a bit cleaner - now shows placeholder if  `showPlaceholder` has been set to true, or `type` has not been overridden